lib/memory.h

```diff
@@ -179,7 +179,7 @@ extern int qmem_walk(qmem_walk_fn *func, void *arg);
 extern int log_memstats(FILE *fp, const char *);
 #define log_memstats_stderr(prefix) log_memstats(stderr, prefix)
 
-extern void memory_oom(size_t size, const char *name);
+extern void memory_oom(size_t size, const char *name) __attribute__((__noreturn__));
```

Comment:
I see we use `__attribute__((__noreturn__))` before the name of the function, can we keep the same style here?

Reply:
Yes, done!